options/glibc/include/bits/glibc/glibc_stdlib.h

```diff
@@ -0,0 +1,19 @@
+#ifndef MLIBC_GLIBC_STDLIB_H
+#define MLIBC_GLIBC_STDLIB_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef __MLIBC_ABI_ONLY
+
+typedef int (*comparison_fn_t) (const void *__a, const void *__b);
```

Comment:
Actually i just noticed this - this is not a function, so it shouldn't be wrapped by the `#ifndef __MLIBC_ABI_ONLY`